Tasks/Common/packaging-common/nuget/NuGetToolRunner2.ts

```diff
@@ -460,6 +489,11 @@ function buildCredentialJson(authInfo: auth.NuGetExtendedAuthInfo): string {
             }
         });
 
+        if (enpointCredentialsJson.endpointCredentials.length < 1) {
+            tl.debug(`None detected.`);
```

Comment:
1. This debug message isn't useful without also looking at the code. Make a little more verbose?
2. I assume this is another bug fix for service endpoints, unrelated to the env var? I don't have the full context but looks fine.

Reply:
We'll first log `Detected external credentials for:`, then loop through the creds and log all endpoints that creds were found. I added this in case no creds were found so the log will say:
`Detected external credentials for:'
`one detected.`

This fix is just making the logs better. Also, returning an empty enpointCredentialsJson object is unnecessary so returning null instead.

Reply:
ah, I see.